webapp/channels/src/i18n/en.json

```diff
@@ -4490,7 +4490,8 @@
   "post_delete.okay": "Okay",
   "post_delete.someone": "Someone deleted the message on which you tried to post a comment.",
   "post_header.update_status": "Update your status",
-  "post_info.actions.noActions": "No Actions currently\nconfigured for this server",
+  "post_info.actions.noActions.details": "configured for this server",
+  "post_info.actions.noActions.intro": "No Actions currently",
```

Comment:
Instead of `intro` and `details`, let's change the ids to `first_line` and `second_line`. That way I feel it will be clearer for translators.